back-end/token/src/test/java/TokenTest.java

```diff
@@ -0,0 +1,41 @@
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.junit.Test;
+import org.functionstream.functions.token.TokenJDBC;
+import org.functionstream.functions.token.GetTokenFunction;
+
+public class TokenTest {
```

Comment:
These unit tests are incorrect. You need to write the unit tests according to the specifications and make sure that the tests are passed.